app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -53,13 +53,7 @@ <h2>Evaluation</h2>
 
         <dt>Status</dt>
         <dd>
-            <span class="badge badge-{{ evaluation.status_context }}">
-                {% if evaluation.animate %}
-                    <span class="spinner-border spinner-border-sm" role="status"
-                          aria-hidden="true"></span>
-                {% endif %}
-                {{ evaluation.get_status_display }}
-            </span>
+            {% include "algorithms/job_status_badge_detail.html" with object=evaluation %}
```

Comment:
Note this is for the evaluation status and not the job status, you have not implemented this yet and should follow with another PR.

Reply:
reverted